### DISCUSSION TITLE
New planet type: Maintenance Sprawl

### DISCUSSION THREAD

**Discussion by swifterjack** [shiptest-ss13/Shiptest#97] [Ideasguy]
You're flying through space, testing your cool and new ships, when on the overmap you see a planetoid that doesn't look similar to the others.
This one has a station icon, and is classed as a "Failed Expansion Planet" (work in progress name)

You dock, and you find a hell of procedurally generated maintenance shafts and small station rooms, with a singular ruin in the center of it. This ruin would be something special, some big deal, as this planetoid only spawns once per system.
The concept for this planet types backstory is similar to the classic paperclip ai one, the AI was tasked to maintain and build a station, perhaps similarly to the drone lawset, and did so infinitely, disregarding the lives of the occupants inside of it, instead creating more metal with more air in space. The AI dies, one way or another, as all things do, and now this flying haunted station is stuck in space, filled to the brim with tiders delights.

**Comment by HighPrincessErinys** [comment]
i am only on board with this if one of the ruins is just a small 3x3 room with a wisdom cow in it as a thorough time waster
